python/src/aiconfig/editor/client/src/contexts/AIConfigContext.tsx

```diff
@@ -7,7 +7,7 @@ import { ClientAIConfig, LogEvent, LogEventData } from "../shared/types";
  */
 const AIConfigContext = createContext<{
   getState: () => ClientAIConfig;
-  logEvent?: (event: LogEvent, data?: LogEventData) => void;
+  logEventHandler?: (event: LogEvent, data?: LogEventData) => void;
```

Comment:
nit, I kind of like the context one just being `logEvent` for succinctness since I usually think of 'handler' suffix referring to a callback prop vs a function from context

Reply:
Oh ok, I really want the function itself to have a different name though than the type event param that it takes in. Otherwise it's not obvious what `logEvent` really means (I know one has capitalized L but still not super clear). Do you have any other suggestions? If so I'll change. If not, I'll just land and we can change later, not a P0